console/ui/src/app/accounts/accounts.component.ts

```diff
@@ -110,16 +110,18 @@ export class AccountListComponent implements OnInit {
   }
 
   deleteAccount(event, i: number, o: ApiUser): void {
-    event.target.disabled = true;
-    event.preventDefault();
-    this.error = '';
-    this.consoleService.deleteAccount('', o.id, false).subscribe(() => {
+    if (confirm('Are you sure to delete ' + o.username + '. All associated data will be lost')) {
```

Comment:
The use of js confirm dialogs is discouraged as they cannot be styled and won't be consistent with the existing confirmation prompts we already have elsewhere (see: https://github.com/heroiclabs/nakama/blob/master/console/ui/src/app/config/config.component.html#L97).